agent/installer/installer_test.go

```diff
@@ -40,6 +40,31 @@ var _ = Describe("Byohost Installer Tests", func() {
 			}
 		})
 	})
+	Context("When installer is created", func() {
```

Comment:
This and the below Contexts read the same. You can choose to either set different description to Contexts (so that they are meaningful for the tests they contain), or if they are really the same, then move the `It` blocks under a single Context

Reply:
done. B)